requirements/requirements.txt

```diff
@@ -0,0 +1,77 @@
+#
```

Comment:
Why are there so many more requirements than before?

Reply:
If you look at `pyproject.toml`, you'll see there different options like `docs`, `tests`, etc. Each of these environments will have their own requirements file; and these are generated automatically by `pip-tools`